subsys/bluetooth/mesh/light_temp_srv.c

```diff
@@ -212,8 +212,6 @@ static void lvl_delta_set(struct bt_mesh_lvl_srv *lvl_srv,
 	 */
 	srv->temp_last = lvl_to_temp(srv, start_value);
 
-	(void)bt_mesh_light_temp_srv_pub(srv, NULL, &status);
```

Comment:
This should not be removed from this set function, but moved to the end. This function causes a change to the light_temp, and should trigger a publish. Same above.

Reply:
I've restored publish functions back.